blog/01-15-23-Build-On-Chain-Puzzle-Game/01-15-23-Build-An-On-Chain-Puzzle-Game-On-Celo.md

```diff
@@ -0,0 +1,300 @@
+---
+title: Build an On-Chain Puzzle Game on Celo
+description: Learn how to build a On-Chain Puzzle Game on Celo using Solidity and Hardhat.
+authors:
+  - name: Nikhil Bhintade
+tags: [celosage, intermediate, hardhat, solidity]
+hide_table_of_contents: true
+slug: "/tutorials/build-an-on-chain-puzzle-game-on-celo"
+---
+
+# Introduction
```

Comment:
**You should add another # hashtag here, to make it a double hashtag**

Reply:
I think I am following previous guideline in this so that is why all the headings are h1 instead of h2. I'll change it soon.